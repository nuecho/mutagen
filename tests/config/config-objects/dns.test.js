importDependencies("dn-dependencies-config.json");
cfgObjectTests("dns", {checkInitialExport: false});
