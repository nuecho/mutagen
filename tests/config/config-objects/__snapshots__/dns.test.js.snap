// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen dns config import-export] should create the dns: the dns have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ dn => number: '123', switch: 'switchForDn', type: 'Extension', name: 'cfgObjectTest', tenant: 'Environment'
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen dns config import-export] should throw exception when new dns to import miss mandatory properties dns: exception has been thrown 1`] = `
Object {
  "stderr": "Cannot import configuration: some configuration objects' mandatory properties for creation are not set.
	Mandatory properties [routeType] not set in dn 'number: '1234', switch: 'switchForDn', type: 'Extension', tenant: 'Environment''
",
  "stdout": "",
}
`;

exports[`[mutagen dns config import-export] should update the modified dns: the modified dns have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ dn => number: '123', switch: 'switchForDn', type: 'Extension', name: 'cfgObjectTest', tenant: 'Environment'
Completed. 1 object(s) imported.
",
}
`;

exports[`[mutagen dns config import-export] should update the same dns: the dns have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ dn => number: '123', switch: 'switchForDn', type: 'Extension', name: 'cfgObjectTest', tenant: 'Environment'
Completed. 1 object(s) imported.
",
}
`;
